supertokens_python/recipe/dashboard/utils.py

```diff
@@ -106,5 +132,95 @@ def get_api_if_matched(path: NormalisedURLPath, method: str) -> Optional[str]:
         return USERS_LIST_GET_API
     if path_str.endswith(USERS_COUNT_API) and method == "get":
         return USERS_COUNT_API
+    if path_str.endswith(USER_API) and method in ("get", "delete", "put"):
+        return USER_API
+    if path_str.endswith(USER_EMAIL_VERIFY_API) and method in ("get", "put"):
+        return USER_EMAIL_VERIFY_API
+    if path_str.endswith(USER_METADATA_API) and method in ("get", "put"):
+        return USER_METADATA_API
+    if path_str.endswith(USER_SESSION_API) and method in ("get", "post"):
+        return USER_SESSION_API
+    if path_str.endswith(USER_PASSWORD_API) and method == "put":
+        return USER_PASSWORD_API
+    if path_str.endswith(USER_EMAIL_VERIFY_TOKEN_API) and method == "post":
+        return USER_EMAIL_VERIFY_TOKEN_API
 
     return None
+
+
+def is_valid_recipe_id(recipe_id: str) -> bool:
+    return recipe_id in ("emailpassword", "thirdparty", "passwordless")
+
+
+async def get_user_for_recipe_id(user_id: str, recipe_id: str) -> Dict[str, Any]:
```

Comment:
The return type should be a proper type and not a key value pair

Reply:
Fixed.